### PR TITLE
Add content

### PR DESCRIPTION
**Pull request by MattiSG**
- Document Attribution.
- Document Detection actors.
- Document video transcription tools.
- Unify Actors pages titles.
- Document a couple other initiatives.

Sorry, this is one big PR instead of several small ones. If this makes it harder to review I can split it.

### CODE REVIEW

File: _pages/encyclopedia/5-attribution/1-case-studies.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+parent: Attribution
+title: Case studies
+nav_order: 1
+permalink: /encyclopedia/Attribution/case-studies
+---
+
+# Attribution case studies
+
+The SSCI US congressional enquiry into the 2016 presidential election is a good example of a long-term, independent, cross-checked investigation and attribution of responsibility.
```
Comment (BerylBogui) [inline _pages/encyclopedia/5-attribution/1-case-studies.md]: > Préciser ce qu'est le Social Sciences Citation Index

File: _pages/encyclopedia/5-attribution/index.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+permalink: /encyclopedia/Attribution
+---
+
+# Attribution
+
+The phase in which responsibility is assigned to some entity for an information manipulation campaign.
+
+After some content has matched all [criteria](concepts/criteria) in the [Qualification](qualification) phase, investigation and forensics can be used to attribute responsibility.
+
+Attribution is a difficult business, as false hints can be left to impersonate and mislead a public reaction to the wrong entity. It would be trivial, for example, to buy artificial followers to inflate some target’s audience without their assent or even knowledge, in order to get them accused of then artificially boosting their message.
```
Comment (BerylBogui) [inline _pages/encyclopedia/5-attribution/index.md]: To clarify: 
Attribution is a difficult business, as false hints can be left for impersonation purposes, misleading the public into reacting to the wrong entity.

Comment (BerylBogui) [inline _pages/encyclopedia/5-attribution/index.md]: Not sure what you mean using "trivial" in this context

Also to clarify: "to buy artificial followers to inflate the audience of a target without its assent or even knowledge, in order to get the target accused of artificially boosting a message".

File: _pages/encyclopedia/6-prevention/3-tools.md [hunk 3]
```diff
@@ -13,6 +13,15 @@ permalink: /encyclopedia/prevention/tools
 
 - - -
 
+## For public institutions
+
+### [UK Government Communication Service RESIST toolkit](https://gcs.civilservice.gov.uk/wp-content/uploads/2019/03/RESIST_Toolkit.pdf)
+
+The RESIST toolkit is a 50-pager full of insights and actionable tools for public service officials to enable understanding and preparation for disinformation campaigns. It also contains thorough descriptions of tactics used by attackers.
```
Comment (BerylBogui) [inline _pages/encyclopedia/6-prevention/3-tools.md]: To clarify: 
The RESIST toolkit is a 50-pager full of insights and actionable tools to enable public service officials to understand and prepare for disinformation campaigns.

File: _pages/encyclopedia/4-reaction/2-practices.md [hunk 4]
```diff
@@ -17,6 +17,11 @@ For instance, in March 2019, several Algerian media outlets (including _observal
 
 It is difficult to assess the efficiency of such denials but direct reactions can help discredit disinformation-spreading outlets, instances and individuals as well as facilitate the work and action of fact-checkers.
 
+### Chatbots
+
+In Taiwan, a developer created [Aunt Meiyu](https://g0v.news/auntmeiyu-70294724df57), a bot that can be included in private chats and that will flag fake news and rumors as they are shared within family and friends groups. Over a hundred thousand users have set up this bot, with about 3% block rate.
```
Comment (BerylBogui) [inline _pages/encyclopedia/4-reaction/2-practices.md]: Grammar: 
* with a block rate of circa 3%

File: _pages/encyclopedia/2-detection/3-tools.md [hunk 5]
```diff
@@ -120,3 +120,12 @@ Trendolizer's API allows subscribers to extract all information into JSON-format
 
 The creator of Trendolizer, Maarten Schenk, is also the co-founder of Lead Stories, a online fact-checking and debunking media outlet which uses the Trendolizer engine to detect and further analyze trending news. A sample of their work can be found [here](https://hoax-alert.leadstories.com/3470102-how-a-tattoo-led-us-to-the-macedonian-trucker-delivering-loads-of-fake-news-to-the-internet.html). It shows how Trendolizer can actually be used in fighting disinformation.
 
+
+## Criterion: Harmful
+
+### Automatic video transcription and annotation
+
+For video content, the first difficulty is in making content discoverable without relying on watching the entirety of all streams. Two initiatives supported by the [EU Media Hub](http://www.europarl.europa.eu/stoa/en/media-hub) transcribe audio streams and annotate the video streams with object and personalities recognition.
```
Comment (BerylBogui) [inline _pages/encyclopedia/2-detection/3-tools.md]: Grammar and clarity:
"difficulty is to make content discoverable without relying on the watching of all streams in their entirety".

File: _pages/encyclopedia/5-attribution/1-case-studies.md (hunk 1 above)
Comment (MattiSG) [reply _pages/encyclopedia/5-attribution/1-case-studies.md]: Mmmmh no, that's not Social Sciences Citation Index but [Senate Select Committee on Intelligence](https://www.intelligence.senate.gov) 😅 So might indeed be useful to add a link.

File: _pages/encyclopedia/2-detection/3-tools.md (hunk 5 above)
Comment (MattiSG) [reply _pages/encyclopedia/2-detection/3-tools.md]: Not sure of “relying on _the_ watching of”. Otherwise go for it.

**Comment by MattiSG** [comment]
Ok great please apply those changes and let's ship!  😄

**Pull request merged**